### ISSUE TITLE
Install Bun with AGS

### ISSUE THREAD

**Issue by Chaitran77** [koeqaife/hyprland-material-you#44]
After install, nothing based on AGS was working (i.e. bar, app menu, cheatsheet). I figured out it was because bun wasn't installed, so I tried installing bun instead of bun-bin which was a time-consuming mistake. Cancelled the install after it downloaded 8GB and got stuck building after that and installed bun-bin which took less than a minute and everything appeared after.

It could also be that AGS wasn't installed with this script/to begin with and I did it manually with paru (can't remember if I did though), which didn't install bun or installed a version that depended on bun but didn't install it.

TLDR: AGS and Bun are required

**Comment by koeqaife** [comment]
try installing with yay

**Comment by zielOS** [comment]
Last time when I was in arch, I tried to install bun with yay but gave me errors. Try to do: yay -S bun-bin.

**Comment by Chaitran77** [comment]
Yep I resorted to the same thing just with Paru instead (my package manager of choice).
